### PR TITLE
RISCV definitions

### PR DESCRIPTION
**Pull request by mmamayka**
Adds
- C primitive types definitions (int, long, etc.)
- RISCV ISA primitive types definitions (Word, Half, Double, etc.)
- Instruction & Operand structure definitions

### CODE REVIEW

File: include/besm-666/instruction.hpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#ifndef BESM666_RISCV_INSTRUCTION_H
+#define BESM666_RISCV_INSTRUCTION_H
```
Comment (c71n93) [inline include/besm-666/instruction.hpp:2]: Why not use `#pragma once`?

File: include/besm-666/riscv-types.hpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#ifndef BESM666_RISCV_TYPES_H
+#define BESM666_RISCV_TYPES_H
```
Comment (c71n93) [inline include/besm-666/riscv-types.hpp:2]: Why not use `#pragma once`?

**Review by c71n93 — COMMENTED** [review]
@mmamayka Also fix clang-format, please

**Review by levBagryansky — DISMISSED** [review]

File: include/besm-666/instruction.hpp (hunk 1 above)
Comment (mmamayka) [reply include/besm-666/instruction.hpp]: Fixed

File: include/besm-666/riscv-types.hpp (hunk 2 above)
Comment (mmamayka) [reply include/besm-666/riscv-types.hpp]: Also fixed

**Review by c71n93 — APPROVED** [review]
Nice!

**Review by levBagryansky — APPROVED** [review]

**Pull request merged**